react-app/src/library/services/vendor-implementations/jabra/jabra-native/jabra-native.ts

```diff
@@ -19,7 +19,8 @@ export default class JabraNativeService extends VendorImplementation {
   ignoreNextOffhookEvent = false;
   _connectionInProgress: any; // { resolve: Function, reject: Function }
   cefSupportsJabra = true;
-  incomingConversationId = ''
+  pendingConversationId: string;
+  activeConversationId: string;
```

Comment:
I don't see where `activeConversationId` is actually set in `jabra-native`. Is it used/needed?